### PR TITLE
feat(eval): champion challenger genops autosxs eval vertex pipelines

### PR DESCRIPTION
**Pull request by agarwal22megha**
# Description

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [x] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [x] You are listed as the author in your notebook or README file.
  - [x] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [x] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [x] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [x] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: gemini/evaluation/champion_challenger_eval_pipeline/README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+``` JSON
+ {
+    "model": "MODEL_NAME",
+    "system_instruction": "Your system isntructions. ",
+    "prompt_template": "Prompt template",
+    "temperature": TEMPERATURE,
+    "max_output_tokens": OUTPUT_TOKENS,
+    "top_p": TOP_P
+}
+```
```
Comment (holtskinner) [inline gemini/evaluation/champion_challenger_eval_pipeline/README.md:61]:
```suggestion
```json
 {
    "model": "MODEL_NAME",
    "system_instruction": "Your system instructions.",
    "prompt_template": "Prompt template",
    "temperature": TEMPERATURE,
    "max_output_tokens": OUTPUT_TOKENS,
    "top_p": TOP_P
}
```
```

**Review by holtskinner — COMMENTED** [review]
@jsondai Can you review this? And/or suggest any improvements that can me made to use the newer eval framework?

**Review by jsondai — APPROVED** [review]
LGTM

**Review by jsondai — APPROVED** [review]

**Review by inardini — COMMENTED** [review]
LGTM. As discussed, consider to stage this in legacy subfolder till you update the AutoSxS component. Thanks

**Review by inardini — APPROVED** [review]

**Pull request merged**